dsl/camel-jbang/camel-jbang-core/src/main/resources/quarkus-dependencies.properties

```diff
@@ -0,0 +1,18 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+
+org.messaginghub\:pooled-jms = io.quarkiverse.messaginghub:quarkus-pooled-jms:2.5.0
```

Comment:
`quarkus-pooled-jms` is managed in `quarkus-camel-bom`, so the version could be empty here?

Reply:
Ah okay, lets try that

Reply:
Yep that works. If you know about other JARs you can add them to the properties file